### ISSUE TITLE
Support for multi-class segmentation

### ISSUE THREAD

**Issue by rose-jinyang** [RElbers/region-mutual-information-pytorch#3]
Hello
How are you?
Thanks for contributing to this project.
I have a question.
Does this repo support the multi-class segmentation?
That's because you used ONLY binary cross entropy & sigmoid in calculating the RMI loss.

**Comment by rose-jinyang** [comment]
Thanks

**Comment by rose-jinyang** [comment]
Hello
How are you?
Thanks for contribution to this project.
I'm NOT sure if this RMI loss would work well in case that we resize the image & mask to input size(NxN in pixels) without keeping width-height ratio in the data augmentation step.
I am working on image segmentation project.
There are many images & masks with different sizes in my dataset.
The data by dataloader are resized to input size(ex: 256x256) and feed into the model.
So the original width-height ratio of image & mask are NOT kept.
Even in such case, does this RMI loss work well?
